src/main/java/com/aspire/student/controller/StudentController.java

```diff
@@ -211,7 +211,8 @@ public void getAllStudents(ResourceRequest request, ResourceResponse response) {
       logger.info("Getting all student list");
       studentList = studentService.getStudentList();
     } catch (Exception e) {
-      logger.error("Error while getting all students", e);
+      //logger.error("Error while getting all students", e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://13.59.219.140:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)